src/Feature/Editor/EditorConfiguration.re

```diff
@@ -238,10 +271,18 @@ let fontFamily =
     ~vim=VimSettings.guifont,
     "editor.fontFamily",
     string,
+    // XXX: Might have to remove the regular part"
     ~default="JetBrainsMono-Regular.ttf",
```

Comment:
Good catch here. This is a bit janky, as we special case the default of `JetBrainsMono-Regular.ttf` - it's sort of a sentinel value for 'load-our-default-font'.

To handle this correctly - we'd have to handle it in `Service_Font`'s loader:

https://github.com/onivim/oni2/blob/0b6e63603632f480280aac054e6a77c04d8af0f1/src/Service/Font/Service_Font.re#L71

And in the case we're using the default font, we'd switch to actually load one of the variants, based on the passed-in fontWeight:
![image](https://user-images.githubusercontent.com/13532591/100009886-4d417300-2d84-11eb-9d89-0db062735628.png)